### PR TITLE
journalEntry and Calendar componets. submiting now to seek asstience …

### PR DESCRIPTION
**Pull request by Jorge1772**
…with rendering the calendar

### CODE REVIEW

**Comment by Jorge1772** [comment]
Calendar dosesnt work yet having issues getting it to work and my journal is commented out till i can connect my calendar to my journal

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@sendgrid/mail": "^7.2.4",
     "bcryptjs": "^2.4.3",
+    "calender": "^1.0.3",
```
Comment (ErnieAtLYD) [inline package.json:9]: @Jorge1772 sanity check me: calender, with an e? not calendar with an a? So you mean you want to install this?
https://github.com/ded/calEnder

File: client/src/components/Calendar.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*import React, { useState } from 'react';
```
Comment (ErnieAtLYD) [inline client/src/components/Calendar.jsx:1]: you know my feelings about leaving in large swaths of comments. :)

Comment (Jorge1772) [reply client/src/components/Calendar.jsx]: @ErnieAtLYD  i just made the changes and pushed up to this branch a few moments ago. please advice if that didnt correct the issue. thank you the comments were removed. i just didnt want to cause any issues when starting the front or backend.

**Pull request closed without merging**